make-util.js

```diff
@@ -1144,7 +1144,7 @@ exports.toOverrideString = toOverrideString;
 
 var createExtension = function(manifest) {
     ensureTool('tsc', '--version', 'Version 3.2.2');
-    ensureTool('mocha', '--version', '5.2.0');
+    ensureTool('mocha', '--version', '9.2.2');
```

Comment:
Could it make sense to extract the version as const? (e.g. `TARGET_MOCHA_VERSION`)